packages/x-charts-pro/src/FunnelChart/FocusedFunnelSection.tsx

```diff
@@ -0,0 +1,112 @@
+'use client';
+import * as React from 'react';
+import { useTheme } from '@mui/material/styles';
+import type { Point } from '@mui/x-charts/internals';
+import { line as d3Line } from '@mui/x-charts-vendor/d3-shape';
+import { useFocusedItem } from '@mui/x-charts/hooks';
+import { useFunnelSeriesContext, useXAxis, useYAxis } from '../hooks';
+import { createPositionGetter } from './coordinateMapper';
+import { getFunnelCurve } from './curves/getFunnelCurve';
+
+export function FocusedFunnelSection(props: React.SVGAttributes<SVGRectElement>) {
+  const theme = useTheme();
+  const focusedItem = useFocusedItem();
+
+  const funnelSeries = useFunnelSeriesContext()?.series?.[focusedItem?.seriesId ?? ''];
```

Comment:
Isn't the empty string technically a valid series ID? 

I feel a bit weird to mention it because it's very unlikely someone will do that, but it could cause weird bugs as well.